src/client/context.rs

```diff
@@ -1445,6 +1450,17 @@ impl Context {
         }
     }
 
+    /// Adds a string to message queue, which is sent joined by a newline
+    /// when context goes out of scope.
+    ///
+    /// **Note**: Only works if the `say` method works.
+    pub fn queue(&mut self, content: &str) -> &mut Self {
+        let to_push = "\n".to_owned() + content;
+        self.queue.push_str(&to_push);
+
+        self
+    }
```

Comment:
This can be faster by pushing the `\n` char and then pushing `content`, avoiding producing a temporary String